python_modules/libraries/dagster-celery-k8s/dagster_celery_k8s_tests/test_executor.py

```diff
@@ -0,0 +1,134 @@
+import subprocess
+from unittest import mock
+
+from dagster import (
+    _check as check,
+    job,
+    op,
+)
+from dagster._core.definitions.reconstruct import ReconstructableJob
+from dagster._core.events.utils import filter_dagster_events_from_cli_logs
+from dagster._core.execution.api import execute_job
+from dagster._core.test_utils import instance_for_test
+from dagster._serdes import serialize_value, unpack_value
+from dagster_celery_k8s.executor import celery_k8s_job_executor
+
+
+@op
```

Comment:
it would be nice if this included a tag that was overridden by the executor config

Reply:
done